platforms/android/assets/www/js/factories.js

```diff
@@ -670,12 +1041,62 @@ angular.module('app.factories', [])
   })
 
   // $firebaseAuth instance
-  .factory('Auth', ['$firebaseAuth',
+  .factory('Auth', ['$firebaseAuth', 
+                    
+
     function ($firebaseAuth) {
-      return $firebaseAuth();
+        return $firebaseAuth();
     }
+    
   ])
 
+
+
+
+  // $firebaseUser instance
+  .factory('User', ['$firebaseAuth',
```

Comment:
You can inject the `Auth` factory and return `Auth.$getAuth()`, which is the user. Or you can create you own user object and pass in the properties found in  `Auth.$getAuth()`.